### ISSUE TITLE
3 tests fail

### ISSUE THREAD

**Issue by yurivict** [dwavesystems/dwave-preprocessing#44]
```
======================================================================
ERROR: test_ignored_interactions (test_scale.TestScaleComposite)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/dwave-preprocessing-0.4.0/tests/test_scale.py", line 70, in test_ignored_interactions
    dtest.assert_sampleset_energies(sampleset, bqm)
  File "/usr/local/lib/python3.9/site-packages/dimod/testing/asserts.py", line 204, in assert_sampleset_energies
    for sample, energy in sampleset.data(['sample', 'energy']):
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1251, in data
    record = self.record
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1100, in record
    self.resolve()
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1449, in resolve
    samples = self._result_hook(self._future)
  File "/usr/local/lib/python3.9/site-packages/dimod/decorators.py", line 86, in <lambda>
    return SampleSet.from_future(next(iterator), lambda _: next(iterator))
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/stage/usr/local/lib/python3.9/site-packages/dwave/preprocessing/composites/scale.py", line 140, in sample
    sampleset.record.energy = original_bqm.energies(sampleset)
  File "/usr/local/lib/python3.9/site-packages/dimod/binary/binary_quadratic_model.py", line 1092, in energies
    return self.data.energies(samples_like, dtype=dtype)
  File "dimod/binary/cybqm/cybqm_template.pyx.pxi", line 360, in dimod.binary.cybqm.cybqm_float64.cyBQM_template.energies
  File "/usr/local/lib/python3.9/functools.py", line 888, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
TypeError: _as_samples_sampleset() got an unexpected keyword argument 'labels_type'

======================================================================
ERROR: test_ignored_offset (test_scale.TestScaleComposite)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/dwave-preprocessing-0.4.0/tests/test_scale.py", line 85, in test_ignored_offset
    dtest.assert_sampleset_energies(sampleset, bqm)
  File "/usr/local/lib/python3.9/site-packages/dimod/testing/asserts.py", line 204, in assert_sampleset_energies
    for sample, energy in sampleset.data(['sample', 'energy']):
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1251, in data
    record = self.record
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1100, in record
    self.resolve()
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1449, in resolve
    samples = self._result_hook(self._future)
  File "/usr/local/lib/python3.9/site-packages/dimod/decorators.py", line 86, in <lambda>
    return SampleSet.from_future(next(iterator), lambda _: next(iterator))
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/stage/usr/local/lib/python3.9/site-packages/dwave/preprocessing/composites/scale.py", line 140, in sample
    sampleset.record.energy = original_bqm.energies(sampleset)
  File "/usr/local/lib/python3.9/site-packages/dimod/binary/binary_quadratic_model.py", line 1092, in energies
    return self.data.energies(samples_like, dtype=dtype)
  File "dimod/binary/cybqm/cybqm_template.pyx.pxi", line 360, in dimod.binary.cybqm.cybqm_float64.cyBQM_template.energies
  File "/usr/local/lib/python3.9/functools.py", line 888, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
TypeError: _as_samples_sampleset() got an unexpected keyword argument 'labels_type'

======================================================================
ERROR: test_ignored_variables (test_scale.TestScaleComposite)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/dwave-preprocessing-0.4.0/tests/test_scale.py", line 100, in test_ignored_variables
    dtest.assert_sampleset_energies(sampleset, bqm)
  File "/usr/local/lib/python3.9/site-packages/dimod/testing/asserts.py", line 204, in assert_sampleset_energies
    for sample, energy in sampleset.data(['sample', 'energy']):
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1251, in data
    record = self.record
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1100, in record
    self.resolve()
  File "/usr/local/lib/python3.9/site-packages/dimod/sampleset.py", line 1449, in resolve
    samples = self._result_hook(self._future)
  File "/usr/local/lib/python3.9/site-packages/dimod/decorators.py", line 86, in <lambda>
    return SampleSet.from_future(next(iterator), lambda _: next(iterator))
  File "/disk-samsung/freebsd-ports/science/py-preprocessing/work-py39/stage/usr/local/lib/python3.9/site-packages/dwave/preprocessing/composites/scale.py", line 140, in sample
    sampleset.record.energy = original_bqm.energies(sampleset)
  File "/usr/local/lib/python3.9/site-packages/dimod/binary/binary_quadratic_model.py", line 1092, in energies
    return self.data.energies(samples_like, dtype=dtype)
  File "dimod/binary/cybqm/cybqm_template.pyx.pxi", line 360, in dimod.binary.cybqm.cybqm_float64.cyBQM_template.energies
  File "/usr/local/lib/python3.9/functools.py", line 888, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
TypeError: _as_samples_sampleset() got an unexpected keyword argument 'labels_type'

----------------------------------------------------------------------
Ran 326 tests in 0.733s

FAILED (errors=3)
```

Version: 0.4.0
Python 3.9
FreeBSD 13.1

**Comment by arcondello** [comment]
This is a known issue, resolved by https://github.com/dwavesystems/dimod/pull/1230. It will be fixed by the next dimod deploy. For now the fix is to downgrade dimod to 0.11.3.

**Comment by arcondello** [comment]
Closing this issue because https://github.com/dwavesystems/dimod/pull/1230 was deployed as part of [dimod 0.11.5](https://github.com/dwavesystems/dimod/releases/tag/0.11.5).
